src/common/Smi.Common/Options/GlobalOptions.cs

```diff
@@ -486,7 +486,15 @@ public class DicomAnonymiserOptions : IOptions
         public string? RoutingKeySuccess { get; set; }
         public string? RoutingKeyFailure { get; set; }
         public bool FailIfSourceWriteable { get; set; } = true;
-
+        public string? VirtualEnvPath { get; set; }
+        public string? ShellScriptPath { get; set; }
+        public string? DicomPixelAnonPath { get; set; }
+        public string? SmiServicesPath { get; set; }
+        public string? CtpJarPath { get; set; }
```

Comment:
Maybe clarify this to `CtpAnonCliJar` ?

Reply:
Action: (da) CtpJarPath has been renamed to CtpAnonCliJar